landr.conf.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
       {
         'title': 'True container deltas',
         'image': 'bandwidth',
-        'description': 'Bandwidth-efficient updates with binary diffs, 10-70x smaller than pulling layers in <a href="https://resin.io/blog/announcing-balena-a-moby-based-container-engine-for-iot/#technical_comparison" target="_blank">common scenarios</a>'
+        'description': 'Bandwidth-efficient updates with binary diffs, 10-70x smaller than pulling layers in <a href="https://balena.io/blog/announcing-balena-a-moby-based-container-engine-for-iot/#technical_comparison" target="_blank">common scenarios</a>'
```

Comment:
The domain changes from `resin.io` to `balena.io`, but the blog should still be found at the same path, right?

Reply:
right